README.md

```diff
@@ -1 +1,19 @@
-# hw01
\ No newline at end of file
+# MACS 20500: Homework 1  
+## James Zimmerman  
+
+  
+### Intro  
+Hi, I'm James Zimmerman! I'm a rising third year here at UChicago majoring in **Public Policy** and in **Economics**.
+
+### Facts About Me  
+  + I'm from **New York City**, living my entire life in the same apartment before leaving for college.  
+  + I've become really interested in studying **film** over my second year. I can enjoy any movie that isn't horror, no matter how good or bad.  
+  + I also enjoy **history**, particularly Reconstruction and the 20th-century United States.  
+  + I played basketball and baseball growing up. Today, I too closely follow the **NBA** and the Pittsburgh Steelers.  
+
+### Reflections  
+I did not find this assignment itself to be particularly difficult, as I had previous practice editing files in R Markdown. The tricky bit was learning to use GitHub to clone and then pull request the file. I hope this works!
+
+### Conclusion  
+I end with a picture of New York City at sunrise.
+![Taken near end of senior year.](Sunrise.jpg)
```

Comment:
Great job with the image add here! FYI, for some reason, GitHub won’t show the captions you add to images/tables. I am not sure why this is the case with GitHub because the captions do show up in other formats (like RStudio previews).